client/styles/globals.css

```diff
@@ -0,0 +1,16 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
```

Comment:
do we really need to inject so many fonts? 
I would say add 1 of your choice and 1 default.

Reply:
@ilakshay14  These are default styling of next-boilerplate. I'll be removing all default styles in next update.